webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -1616,7 +1555,7 @@ class StringFormatter private (
    *
    * @param iri the IRI to be checked.
    */
-  def isKnoraPermissionIriStr(iri: IRI): Boolean =
+  def isKnoraPermissionIriStr(iri: IRI): Boolean = // V2 / value objects
```

Comment:
I guess the comments in this file are there for later?

Reply:
Good guess :)